snapcraft/models/project.py

```diff
@@ -805,139 +1237,439 @@ class Project(models.Project):
+        :header-rows: 1
+
+        * - Value
+          - Description
+        * - ``xz``
+          - Default. Use `XZ <https://en.wikipedia.org/wiki/XZ_Utils>`_ compression.
+        * - ``lzo``
+          - Use `LZO <https://en.wikipedia.org/wiki/Lempel%E2%80%93Ziv%E2%80%93Oberhumer>`_ compression.
+
+    """
+
     version: VersionStr | None = pydantic.Field(
         default=None,
         description="The version of the snap.",
-        examples=["version: 1.2.3"],
+        examples=["1.2.3"],
     )
+    """The version of the snap.
+
+    This field is required unless version information is provided by the
+    ``adopt-info`` key.
+
+    See `Using external
+    metadata <https://snapcraft.io/docs/using-external-metadata>`_ for more
+    information.
+    """
+
     donation: UniqueList[str] | None = pydantic.Field(
         default=None,
         description="The snap's donation links.",
-        examples=["donation: donate@example.com, https://example.com/donate"],
+        examples=["donate@example.com, https://example.com/donate"],
     )
+    """The snap's donation links.
+
+    Donation links can be adopted from appstream metadata files.
+
+    See `Using external
+    metadata <https://snapcraft.io/docs/using-external-metadata>`_ for more
+    information.
+    """
+
     # snapcraft's `source_code` is more general than craft-application
     source_code: UniqueList[str] | None = pydantic.Field(  # type: ignore[assignment]
         default=None,
         description="The links to the source code of the snap or the original project.",
-        examples=["source-code: https://example.com/source-code"],
+        examples=["https://example.com/source-code"],
     )
+    """The links to the source code of the snap or the original product.
+
+    Source code links can be adopted from appstream metadata files.
+
+    See `using external
+    metadata <https://snapcraft.io/docs/using-external-metadata>`_ for more
+    information.
+    """
+
     contact: UniqueList[str] | None = pydantic.Field(  # type: ignore[reportIncompatibleVariableOverride]
         default=None,
         description="The snap author's contact links and email addresses.",
-        examples=["contact: [contact@example.com, https://example.com/contact"],
+        examples=["contact@example.com, https://example.com/contact"],
     )
+    """The snap author's contact links and email addresses.
+
+    Contact information can be adopted from appstream metadata files.
+
+    See `using external
+    metadata <https://snapcraft.io/docs/using-external-metadata>`_ for more
+    information.
+    """
+
     issues: UniqueList[str] | None = pydantic.Field(  # type: ignore[reportIncompatibleVariableOverride]
         default=None,
         description="The links and email addresses for submitting issues, bugs, and feature requests.",
-        examples=["issues: issues@email.com, https://example.com/issues"],
+        examples=["issues@email.com, https://example.com/issues"],
```

Comment:
I think this needs to be tweaked for all the url keys (issues, websites, contact, etc)

```suggestion
        examples=["[issues@email.com, https://example.com/issues]"],
```

Reply:
I believe I fixed all of them.

https://github.com/canonical/snapcraft/pull/5421/commits/7cbca1d58702e90b48018f818a03a31f4d2cd1d9